### DISCUSSION TITLE
Error: QWeb2: Template 'FormView.marker_edit_button' not found

### DISCUSSION THREAD

**Discussion by Stephanisk** [gityopie/odoo-addons#159] [General]
hi, 

ik got this error on odoo14 installing the web_google_maps app. Very sad, i would love to try it!

**Comment by gityopie** [comment]
Hi @Stephanisk 
I couldn't reproduce the issue on my side.
maybe better if you provide me your details or steps how you setup the module?

Btw, this is not the right place to report bug or issue :)
